migrate/playbooks/rds-timescale-live-migration.md

```diff
@@ -0,0 +1,410 @@
+```
+
+5. To allow an EC2 instance to connect to an RDS instance, you need to modify the
+security group associated with your RDS instance.
+* Note the Private IPv4 address of your EC2 instance.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/note-private-ipv4.jpg"
+ alt="Note the private IPv4 address"/>
+
+* Go to the security group associated with your RDS instance and select "Edit inbound rules".
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/edit-security-group.jpg"
+ alt="Edit the security group"/>
+
+* Click "Add rule". In the "Type" field, select "PostgreSQL". For "Source", select
+"Custom" and input the Private IPv4 address of your EC2 instance. Add a suitable
+description for this rule. Finally, click on "Save rules" to apply the changes.
+<img class="main-content__illustration"
+ src="https://assets.timescale.com/docs/images/playbooks/rds-to-ts-live-migration/add-new-rule.jpg"
+ alt="Add new rule"/>
+
+6. Verify the connection to the RDS service from your EC2 instance. Please note,
+you will be prompted to enter the master password. This should be the same password
+that you used when creating your AWS RDS service.
+```sh
+# psql -h <rds_endpoint> -p <rds_port> -U postgres -c "select 1"
+psql -h aws-rds-low-downtime-migration-db.c8tzn206yp6f.us-west-2.rds.amazonaws.com -p 5432 -U postgres -c "select 1"
+Password for user postgres:
+ ?column?
+----------
+        1
+(1 row)
+```
+
+## Performing data migration
+<GettingHelp />
+
+The migration process consists of the following steps:
+1. Set up a target database instance in Timescale.
+1. Prepare the source database for the live migration.
+1. Set up a replication slot and snapshot.
+1. Migrate roles and schema from source to target.
+1. Follow the remaining steps from "Live migration" documentation.
+
+<StepOne />
+
+## 2. Prepare the source database for the live migration
+In order to replicate `UPDATE` and `DELETE` operations on tables in the source database,
+the tables must have either a primary key or `REPLICA IDENTITY`. Replica identity
+assists logical decoding in identifying the rows being modified. It defaults to
+using the table's primary key.
+
+If a table doesn't have a primary key, you'll have to manually set the replica identity.
+One option is to use a unique, non-partial, non-deferrable index that includes only
+columns marked as `NOT NULL`.
+
+```sql
+ALTER TABLE {table_name} REPLICA IDENTITY USING INDEX {_index_name};
+```
+
+If there's no primary key or viable unique index to use, you will have to set
+`REPLICA IDENTITY` to `FULL`. If you are expecting a large number of `UPDATE` or `DELETE`
+operations on the table, we do not recommend using `FULL`. For each `UPDATE` or `DELETE`
+statement, Postgres will have to read the whole table to find all matching rows, which
+will result in significantly slower replication.
```

Comment:
Change "significantly slower replication" to something like "the replication will be slow". Using the phrase "significantly low" can demotivate users from utilizing this process.